core/scss/components/input/input.scss

```diff
@@ -1,21 +1,15 @@
 /** @component input */
 
-@import '../forms/settings';
-@import 'mixins';
-@import '../../tools/functions/core';
-
 @include exports('md-input') {
   .#{$input__class} {
-    @include input-color(
-      $color: $input__font-color,
+    @include input-color($color: $input__font-color,
```

Comment:
Why are these arguments/variables moving to first line?

Reply:
must be the linter.